apps/nextjs/src/app/[locale]/manage/about/page.tsx

```diff
@@ -57,18 +62,54 @@ export default async function AboutPage({ params: { locale } }: PageProps) {
       </Center>
       <Text mb="xl">{t("text")}</Text>
 
-      <Accordion classNames={classes}>
+      <Accordion defaultValue="contributors" variant="filled" radius="md">
         <AccordionItem value="contributors">
           <AccordionControl icon={<IconUsers size="1rem" />}>
-            {t("accordion.contributors.title")}
+            <Stack gap={0}>
+              <Text>{t("accordion.contributors.title")}</Text>
+              <Text size="sm" c="dimmed">
+                {t("accordion.contributors.subtitle", {
+                  count: contributorsData.length,
+                })}
+              </Text>
+            </Stack>
           </AccordionControl>
-          <AccordionPanel></AccordionPanel>
+          <AccordionPanel>
+            <Flex wrap="wrap" gap="xs">
```

Comment:
Maybe we could justify center so it would have the same spacing on the right / left side

Reply:
It does not look good with center alignment